extensions/wikia/ExactTargetUpdates/hooks/ExactTargetWiki.hooks.php

```diff
@@ -125,36 +117,4 @@ private function getWfVarsTriggeringUpdate() {
 		];
 		return $aWfVarsTriggeringUpdate;
 	}
-
-	/**
-	 * A simple getter for an object of an ExactTargetCreateWikiTask class
-	 * @return  object ExactTargetCreateWikiTask
-	 */
-	private function getExactTargetCreateWikiTask() {
-		return new ExactTargetCreateWikiTask();
-	}
-
-	/**
-	 * A simple getter for an object of an ExactTargetUpdateWikiTask class
-	 * @return  object ExactTargetUpdateWikiTask
-	 */
-	private function getExactTargetUpdateWikiTask() {
-		return new ExactTargetUpdateWikiTask();
-	}
-
-	/**
-	 * A simple getter for an object of an ExactTargetUpdateCityCatMappingTask class
-	 * @return  object ExactTargetUpdateCityCatMappingTask
-	 */
-	private function getExactTargetUpdateCityCatMappingTask() {
-		return new ExactTargetUpdateCityCatMappingTask();
-	}
-
-	/**
-	 * A simple getter for an object of an ExactTargetDeleteWikiTask class
-	 * @return  object ExactTargetDeleteWikiTask
-	 */
-	private function getExactTargetDeleteWikiTask() {
-		return new ExactTargetDeleteWikiTask();
-	}
 }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$aWfVarsTriggeringUpdate". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=php%3AS1488)